docs/example-narratives/submitting-forecasts.md

```diff
@@ -1,28 +1,137 @@
+{: .nb }
+This article assumes some familiarity with HTTP in general and RESTful
+APIs in particular ([background](../articles/trolie-for-ems-and-ot)).
 
-For the purpose of this example, the JSON listing below may be assumed to be in a file called `input.json`.  The input consists of an array by segment, which then contains an array of ratings for each forecast period.  Note this example also cross the Fall daylight savings transition in the central timezone, thus providing an example of dealing with the notorious "duplicate" 1 AM hour:
+
+### Scenario Quick Links
+{:.no_toc}
+
+* toc
+{:toc}
+
+
+
+## Simplified Example: Transmission Owner Sends Forecast with `curl`
+
+If a Transmission Owner is their own Ratings Provider, they must regularly send
+an Ratings Forecast to their Transmission Provider. TROLIE provides the
+[patchRatingForecastProposal](../spec#tag/Rating-Proposals/operation/patchRatingForecastProposal)
+operation for this purpose.
+
+
+Assume the Transmission Owner creates a file called `input.json` containing
+their forecast. An example of the required format for the file is given below.
 
 ```json
 {% include_relative examples/forecast-ratings-proposal-patch.json %}
 ```
 
-## Pushing input.json to TROLIE
-Given input.json, run the following command to send it to the TROLIE server:
+This example also illustrates handling the fall Daylight Savings transition in
+the Central timezone. The format is one of TROLIE's [supported media
+types](../articles/supported-media-types) named
+`application/vnd.trolie.rating-forecast-proposal.v1+json`.
+
+
+### Pushing `input.json` to TROLIE with `curl`
+{:.no_toc}
+
+Given the above `input.json`, run the following command to send it to the TROLIE server:
 
 ```bash
 curl -d @input.json \
 -X PATCH \
 -H "Content-Type: application/vnd.trolie.rating-forecast-proposal.v1+json" \
+-H "Accept: application/vnd.trolie.rating-forecast-proposal-status.v1+json"
 -o output.json \
-"https://trolie.example.com/rating-proposals/forecasts"
+"https://trolie.example.com/rating-proposals/forecast"
+```
+
+If this submission is successful, `output.json` will contain the contents of the
+response from TROLIE. The format of the response is defined by another TROLIE
+media type: `application/vnd.trolie.rating-forecast-proposal-status.v1+json`. An
+example of this response format is given below:
+
+```json
+{% include_relative examples/forecast-ratings-proposal-status-complete.json %}
 ```
 
-On success, output.json will container a copy of the data just uploaded, with additional metadata on update time and status.  
+
+## Invalid Forecasts for Individual Resources Should be Tolerated
+
+The Robustness Principle, or Postel's Law, states: Be conservative in what you
```

Comment:
The way you've phrased this, the whole paragraph belongs in an ADR, not in the examples.  I've learned this from some of our doc writing people, who have complained to me about architects writing install or usage guides.  The people reading the examples sections really don't care about why it works the way it does, they only need to know how.  It's too much.  

You're trying to set up for your example that shows it is possible to break up the proposal into chunks- I would consider rephrasing it to just illustrate its something you can do and how, not why they should do it or why TROLIE supports it.